fonduer/features/structural_features.py

```diff
@@ -1,13 +1,20 @@
 from builtins import str
 
-from fonduer.lf_helpers import (common_ancestor, get_ancestor_class_names,
-                                get_ancestor_id_names, get_ancestor_tag_names,
-                                get_attributes, get_next_sibling_tags,
-                                get_parent_tag, get_prev_sibling_tags, get_tag,
-                                lowest_common_ancestor_depth)
-from fonduer.models import TemporarySpan
-
-FEAT_PRE = 'STR_'
+from fonduer.supervision.lf_helpers import (
+    common_ancestor,
+    get_ancestor_class_names,
+    get_ancestor_id_names,
+    get_ancestor_tag_names,
+    get_attributes,
+    get_next_sibling_tags,
+    get_parent_tag,
+    get_prev_sibling_tags,
+    get_tag,
+    lowest_common_ancestor_depth,
```

Comment:
one extra comma?

Reply:
It's pretty typical to end with the comma on the last item in Python [[1]](https://stackoverflow.com/questions/11597901/why-are-trailing-commas-allowed-in-a-list) as it makes it easier to edit and simplifies diffs.